src/router/index.js

```diff
@@ -0,0 +1,43 @@
+import { createRouter, createWebHistory } from 'vue-router';
+
+
+const routes = [
+  {
+    path: '/',
+    name: 'Home',
+    component: () => import('@/components/pages/LandingPage.vue'),
+  },
+  {
+    path: '/screenings',
+    name: 'Screenings',
+    component: () => import('@/components/views/ScreeningsFilters.vue'),
+  },
+
+  {
+    path: '/movies',
+    name: 'Movies',
+    component: () => import ('@/components/views/MoviesSummary.vue'),
+  },
+/*{
+    path: '/movies/:movieId',
+    name: 'MoviePage',
+    props: true,
+    component: () => import ('@/components/views/MovieCard.vue'),
+  }, */
+  {
+    path: '/:path*',
+    redirect: '/no-match'
+  },
```

Comment:
I'd leave this route definition to be last as it is a `catch all` route - it will get picked up over the `/no-match` route or any other if there would be more routes defined.
Vue router looks for a matching route top to bottom and essentially such `catch all` routes should be defined last